src/seq/mod.rs

```diff
@@ -228,6 +228,7 @@ pub trait SliceRandom {
     /// Shuffle a mutable slice in place.
     ///
     /// For slices of length `n`, complexity is `O(n)`.
+    /// The resulting permutation is picked uniformly from the set of all possible permutations.
```

Comment:
We could add something like this, and maybe link to Wikipedia, but I'm not sure whether this is just more confusing.

```suggestion
    /// The resulting permutation is picked uniformly from the set of all possible permutations (assuming the generator provides enough randomness to cover all permutations).
```

Reply:
Or:

> permutations, assuming a perfect random generator. Note however that pseudo-random generators have limited state size, thus for example a generator with 64-bits of state could not yield all possible permutations of a deck of cards (52! ≈ 2<sup>225.6</sup>).

Reply:
Is that bias observable if I only shuffle a few thousand to million times? Sounds like one would need a really large set of samples for this to possibly make a difference?

Reply:
True; any attempt to use more than a fraction of the period of an RNG can result in observable bias, therefore it does seem an unnecessary clarification.

Reply:
So should I remove it again?

Reply:
I think yes, since the mention of bias could be mis-leading, being no different than any other sequence of RNG calls beyond what is reasonable for the RNG.

Reply:
All right, I reverted to the old state.